package.json

```diff
@@ -67,5 +67,8 @@
     "hooks": {
       "pre-commit": "npm-run-all -ln --parallel lint test"
     }
+  },
+  "dependencies": {
+    "@sentry/browser": "^8.18.0"
```

Comment:
That `@sentry/browser` dep should be installed inside the `jaeger-ui` directory instead of the root workspace.

Reply:
Didn't get you?

Reply:
Oh nvm, got it